### PR TITLE
enh(http) change get to post for more action cmd popup

### PR DESCRIPTION
**Pull request by loiclau**
## Description

As an operator
I want to send an external command on multiples host/services
And the external command is executed a expected

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [x] 19.04.x
- [x] 19.10.x
- [x] 20.04.x
- [x] 20.10.x (master)

<h2> How this pull request can be tested ? </h2>

create services with long name.
launch multi acknowledgement on many services.

## Checklist

- [ ] I followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: www/include/monitoring/objectDetails/xml/hostSendCommand.php [hunk 1]
```diff
@@ -44,16 +44,22 @@
 
 CentreonSession::start(1);
 $centreon = $_SESSION["centreon"];
-if (!isset($_SESSION["centreon"]) || !isset($_GET["host_id"]) || !isset($_GET["cmd"]) || !isset($_GET["actiontype"])) {
+if (
```
Comment (jeremyjaouen) [inline www/include/monitoring/objectDetails/xml/hostSendCommand.php:47]: isset() can take multiple arguments and will return true if they are all set, 
```suggestion
if (!isset($_SESSION["centreon"],$_POST["host_id"],$_POST["cmd"],$_POST["actiontype"])) {
```

File: www/include/monitoring/objectDetails/xml/serviceSendCommand.php [hunk 2]
```diff
@@ -48,24 +48,26 @@
 
 CentreonSession::start(1);
 $centreon = $_SESSION["centreon"];
-if (!isset($_SESSION["centreon"]) ||
-    !isset($_GET["host_id"]) ||
-    !isset($_GET["service_id"]) ||
-    !isset($_GET["cmd"]) ||
-    !isset($_GET["actiontype"])
+if (
```
Comment (jeremyjaouen) [inline www/include/monitoring/objectDetails/xml/serviceSendCommand.php:51]: Same thing here isset() could take multiple arguments to get rid of repetition

File: www/include/monitoring/objectDetails/xml/serviceSendCommand.php [hunk 3]
```diff
@@ -78,29 +74,28 @@
     if (!$centreon->user->access->checkAction($cmd)) {
         exit();
     }
-    if (!$centreon->user->access->checkHost($host_id)) {
+    if (!$centreon->user->access->checkHost($hostId)) {
```
Comment (sc979) [inline www/include/monitoring/objectDetails/xml/serviceSendCommand.php:77]: shouldn't you check if $hostId !== false

File: www/include/monitoring/objectDetails/xml/hostSendCommand.php [hunk 4]
```diff
@@ -65,23 +66,22 @@
     if (!$centreon->user->access->checkAction($cmd)) {
         exit();
     }
-    if (!$centreon->user->access->checkHost($host_id)) {
+    if (!$centreon->user->access->checkHost($hostId)) {
```
Comment (sc979) [inline www/include/monitoring/objectDetails/xml/hostSendCommand.php:69]:
```suggestion
    if ($hostId === false || !$centreon->user->access->checkHost($hostId)) {
```

Comment (sc979) [reply www/include/monitoring/objectDetails/xml/hostSendCommand.php:69]: or modify checkHost to test it more globally

**Pull request merged**